monkey/common/common_consts/credentials_type.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class CredentialsType(Enum):
```

Comment:
I'm not sure about the change from singular to plural here. In my understanding, credentials (plural) are made up of components, i.e. identities and secrets. A single identity or a single secret would be singular, not plural. Example: "What type of credential is this? It's a password."

We could consider changing this to `CredentialComponentType` to avoid ambiguity.